web/shared/src/internal_url.ts

```diff
@@ -1,4 +1,6 @@
-const hashReplacements = new Map([
+type GetStreamNameFunction = (stream_id: number | string) => string;
```

Comment:
`stream_id` should always be a number. But these arguments are called `maybe_get_stream_name` for a reason: they’re allowed to return `undefined`.